frontends/web/src/locales/en/app.json

```diff
@@ -616,6 +639,7 @@
       "newVersion": {
         "label": "Available version"
       },
+      "requireUpdate": "Requires firmware update",
```

Comment:
What about bitbox02Wizard.advanced.outOfDate (`"Firmware out of date for this feature"`)? Seems redundant.

Reply:
changed `bitbox02Wizard.advanced.outOfDate` to "Requires firmware update" in locize. This was only used to advanced setup when changing to 12 words or skip-sd-card.